apps/src/JSInterpreter.js

```diff
@@ -585,6 +591,97 @@ JSInterpreter.prototype.executeInterpreter = function (firstStep, runUntilCallba
 };
 
 /**
+ * Checks to the see if the supplied node is from the user code range.
+ *
+ * @param {!Object} node supplied by acorn parse.
+ * @return {boolean} true if the node is in user code.
+ * @private
+ */
+JSInterpreter.prototype.isNodeInUserCode_ = function (node) {
+  if (typeof node.start === 'undefined' || typeof this.codeInfo === 'undefined') {
+    return false;
+  }
+  var start = node.start - this.codeInfo.userCodeStartOffset;
+
+  return start >= 0 && start < this.codeInfo.userCodeLength;
+};
```

Comment:
It looks like the only thing we use from `node` is `node.start`. Would it make more sense to just have this method take `node.start` as an input (and call it something like `isOffsetInUserCode_`?